### DISCUSSION TITLE
Can dynamic lines be added?

### DISCUSSION THREAD

**Discussion by hl0709** [jgraph/drawio#2128] [Ideas]
Make the lines look flowing, this is actually very useful, making the picture look more dynamic

**Comment by Peeeet** [comment]
If you set the "Flow Animation" property on a connection line, it will show a "marching ants" style of animation.
![image](https://user-images.githubusercontent.com/5926263/147605109-9bf7667c-6572-4b67-b3f7-599124ff35e2.png)
